### PR TITLE
Add Rambox 0.3.0

### PR DESCRIPTION
**Pull request by szhu**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/rambox.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'rambox' do
+  version '0.3.0'
+  sha256 '64d0027f478ddb138862be1e42a2681228c9b451ed52ae8634a4cf7c2ae5d8bd'
+
+  # github.com was verified as official when first introduced to the cask
+  url "https://github.com/saenzramiro/rambox/releases/download/#{version}/Rambox-#{version}-mac.zip"
```
Comment (vitorgalvao) [inline Casks/rambox.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

Comment (vitorgalvao) [inline Casks/rambox.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

File: Casks/rambox.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask 'rambox' do
+  version '0.3.0'
+  sha256 '64d0027f478ddb138862be1e42a2681228c9b451ed52ae8634a4cf7c2ae5d8bd'
+
+  # github.com was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/rambox.rb]: Needs to be more specific: `github.com/saenzramiro/rambox`.

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 4e0641f13922d91f886776689c801f3bcd1aeb51. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by szhu** [comment]
That was amazingly fast. Thank you!

**Pull request closed without merging**